contracts/Rollup.sol

```diff
@@ -299,6 +300,14 @@ contract Rollup is RollupHelpers {
         bytes32 accountRoot = accountRegistry.root();
         bytes32 bodyRoot;
         for (uint256 i = 0; i < stateRoots.length; i++) {
+            require(
```

Comment:
Can we instead do this check in fraud-proof and slash appropriately?

Reply:
Sounds good, will fix.